### DISCUSSION TITLE
Build error when compiling alpine component

### DISCUSSION THREAD

**Discussion by LogicSatinn** [filamentphp/filament#14866] [Help]
### Package

Panel builder

### Package Version

v3.2.49 

### How can we help you?

I'm working on a custom file upload implementation using Filament PHP with an Alpine component. When I compile the component using esbuild, the build fails if I import the Axios library, but passes when I remove it.

Breakdown:
- Importing Axios causes the esbuild compilation to fail
- Removing Axios library allows successful compilation

I absolutely need to use Axios in this component. Has anyone encountered a similar issue or know how to resolve this esbuild configuration conflict?

Attempted Troubleshooting:
- Verified Axios is installed via npm
- Confirmed no syntax errors in the import

Any guidance on resolving this would be greatly appreciated!

Here are the logs to the failed build
[failed esbuild logs.txt](https://github.com/user-attachments/files/17868181/failed.esbuild.logs.txt)

**Comment by leandrocfe** [comment]
Could you share the code that you are using?

**Comment by LogicSatinn** [comment]
Hey @leandrocfe, I apologize for the late reply. I ended up using a different way.
